webpack.common.js

```diff
@@ -10,9 +14,18 @@ module.exports = {
         ]
     },
     plugins: [
+        new CleanWebpackPlugin(),
         new CopyPlugin([{
             from: "**/*",
             to: "."
-        }])
+        }]),
+        new CopyPlugin(
+            Assets.map(asset => {
+              return {
+                from: path.resolve(__dirname, `./node_modules/${asset}`),
+                to: path.resolve(__dirname, './dist/bower_components')
+              };
+            })
+          )
```

Comment:
Something is weird with the indentation here...

Reply:
https://github.com/jellyfin/jellyfin-web/pull/549/commits/651a53d64a2d44abaa7bfd9bf1bf58e5a7fe4122